Confus/Confus/GUI.cpp

```diff
@@ -0,0 +1,47 @@
+#include <Irrlicht/irrlicht.h>
+
+#include "GUI.h"
+
+namespace Confus
+{
+	GUI::GUI(irr::IrrlichtDevice* a_Device, Player* a_Player)
+	{
+		m_GUIEnvironment = a_Device->getGUIEnvironment();
+
+		m_Driver = a_Device->getVideoDriver();
+		m_BloodImage = m_Driver->getTexture("Media/Textures/Blood.png");
+		m_BloodOverlay = m_GUIEnvironment->addImage(m_BloodImage, irr::core::position2d<int>(0, 0));
+		m_BloodOverlay->setVisible(false);
+		m_PlayerNode = a_Player;
+		m_HealthTextBox = m_GUIEnvironment->addStaticText(L"", irr::core::rect<irr::s32>(10, 10, 100, 25), false);
+		m_AudioSourceLowHealth = new OpenALSource("heartbeat.wav");
+	}
+
+	void GUI::update()
+	{
+		m_HealthString = "";
+		m_HealthString += L"Health: ";
+		m_HealthString += m_PlayerNode->PlayerHealth.getHealth();
+		m_HealthString += L"%";
```

Comment:
Yeah seems a bit more to the point, though the outer most parenthesis aren't really necessary